testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -310,16 +300,11 @@ export default PageView.extend({
 
   feedback(evt) {
     evt.preventDefault();
-    const installed = Object.keys(app.me.installed);
-    const queryParams = querystring.stringify({ref: 'givefeedback',
-                                               experiment: this.model.title,
-                                               installed: installed});
-
     app.sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
       eventLabel: 'Give Feedback',
-      outboundURL: `${this.model.survey_url}?${queryParams}`
+      outboundURL: this.model.buildSurveyURL('givefeedback')
```

Comment:
Surveys should open in a new tab.

For reference, this is how the `experiment-page.js` interaction used to work before it was over-written by #726.